metadata/pkg/service/service.go

```diff
@@ -104,29 +107,35 @@ func (f *metadataService) SyncMetadata(ctx context.Context, in *pb.SyncMetadataR
 
 	return nil
 }
-
 func (f *metadataService) ListMetadata(ctx context.Context, in *pb.ListMetadataRequest, out *pb.ListMetadataResponse) error {
-	log.Info("Received GetMetadata request in metadata service.")
-	res, err := db.DbAdapter.ListMetadata(ctx, in.Limit)
-	if err != nil {
-		log.Errorf("Failed to create backend: %v", err)
+	log.Info("received GetMetadata request in metadata service.")
```

Comment:
better to add info for example:
log.Infof("received GetMetadata request in metadata service:%+v", in)

Reply:
![image](https://user-images.githubusercontent.com/48655639/214242572-49059b4d-4e97-4597-af82-4835d0df6a08.png)

done